src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SupplementaryEvidenceTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator;
+
+import com.microsoft.azure.servicebus.Message;
+import org.awaitility.Duration;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.TestHelper;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.ScannedDocument;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.EnvelopeParser;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Document;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertEquals;
+
+public class SupplementaryEvidenceTest extends BaseTest {
+
+    @Test
+    public void should_attach_supplementary_evidence_to_the_case() throws Exception {
+        // given
+        String caseData = SampleData.fromFile("envelopes/new-envelope.json");
+        Envelope envelope = EnvelopeParser.parse(caseData);
+        CaseDetails caseDetails = ccdCaseCreator.createCasee(envelope);
+        Assert.assertNotNull(caseDetails.getId());
+
+        // when
+        JSONObject updateData = new JSONObject(SampleData.fromFile("envelopes/update-envelope.json"));
+        updateData.put("case_ref", caseDetails.getId());
+        byte[] updatedEnvelopeData = updateData.toString().getBytes();
+
+        Message message = new Message();
```

Comment:
create method `createMessage` it makes it read better (also you can just pass in the file name)

Reply:
Id suggest a tiny bit of kotlin code to decorate the QueueClient would simplify this.

```kotlin 
class ScanningMessager(@Autowired private val queueClient: QueueClient): IQueueClient by queueClient
{
    fun sendMessageFromFile( fileName:String, caseRef:Long ){
        // when
        val updateData = JSONObject(SampleData.fromFile("envelopes/update-envelope.json"))
        updateData.put("case_ref", caseRef)
        val updatedEnvelopeData = updateData.toString().toByteArray()

        val message = Message()
        message.messageId = UUID.randomUUID().toString()
        message.body = updatedEnvelopeData
        queueClient.send(message)
    }
}
```
Or if you convert this file to kotlin it becomes even simpler
```kotlin
fun QueueClient.sendMessageFromFile( fileName:String, caseRef:Long ){
        // when
        val updateData = JSONObject(SampleData.fromFile("envelopes/update-envelope.json"))
        updateData.put("case_ref", caseRef)
        val updatedEnvelopeData = updateData.toString().toByteArray()

        val message = Message()
        message.messageId = UUID.randomUUID().toString()
        message.body = updatedEnvelopeData
        queueClient.send(message)
    }
```